### ISSUE TITLE
"Ignored Updates" list

### ISSUE THREAD

**Issue by ryanaltvater** [atom/atom#15319]
I don't know if anyone else has suggested this, but it would be really nice in addition to "Available Updates" for packages if there was also an "Ignored Updates" list as well.

I've had this happen a few times before, but today I ran into an issue with a package where I had to manually install an older version. Now every time I open Atom, I'm notified of an available update for that package, which I don't want to update. I do, however, want to update other packages but can't utilize the "Update All" button to update them all at once. I'll have to manually click "Update" on each individual package just to not have the older version package accidentally update. It'd be nice to be able to "ignore update" and then it adds itself to an "Ignored Updates" list just below "Available Updates". I wouldn't receive update notifications for packages on this new list, and in the future if I wanted to update packages on this list I could just remove them from the list and they would move back to the "Available Updates" list for updating.

**Comment by 50Wliu** [comment]
Use the newly available `core.versionPinnedPackages` config setting.

**Comment by ryanaltvater** [comment]
@50Wliu Is there documentation how to do this? This is all I've found. https://github.com/atom/atom/pull/14417 Be nice if there was a GUI to managing this, if you currently have to open a file and edit it manually (just assuming since I can't find any documentation how to implement pinned packages exactly). Thank you!

**Comment by 50Wliu** [comment]
Yeah, it should look like this in Settings View:
![version-pinned-packages](https://user-images.githubusercontent.com/2766036/29338353-d01a9a3c-81e2-11e7-935d-4cc89d0312a2.png)
Syntax should look like `package1, package2, package3`

**Comment by ryanaltvater** [comment]
Excellent! That’s awesome. Did not know it was implemented like that. Exactly what I’m looking for. Thanks again! 😊

Ryan

**Comment by ryanaltvater** [comment]
@50Wliu So a little bit of an update since using the new "Version Pinned Packages". I'm finding that even though I'm ignoring updates for a specific package, somehow after I shut down Atom and open it back up, that package randomly gets updated on it's own. Within the last 24 hours, I've had to uninstall the package, and reinstall a specific version. This morning, I opened Atom and noticed the package had been updated. Now, I just shut Atom down again and when I opened it back up, the package was updated once again. Seems to have started happening within the last few days, after I updated Atom to the latest version. Any ideas? Maybe a hiccup with the latest release of Atom? :(
